### PR TITLE
HW1 - Victor Vazquez

### PR DESCRIPTION
**Pull request by victorhugovazquez**
Hi, it was not a difficult assignment, but I did not know the code for links and pictures so I googled some webpages to learn how to do it. I found it useful to future projects like it looks more professional than just simple copy-pasting the code in a word sheet. Victor.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,16 @@
-# hw01
\ No newline at end of file
+#### Biography (HW1)
```
Comment (shharrison) [inline README.md:1]: Good use of header.

File: README.md [hunk 2]
```diff
@@ -1 +1,16 @@
-# hw01
\ No newline at end of file
+#### Biography (HW1) 
+
+Hi, 
+
+My name is **Victor Vazquez**. I am a second year MPP student from ~~Haris Schul~~, sorry, from Harris School of Public Policy. I have work experience in the private sector, especially in the banking sector. I am a current member of the organizations *Latin American Matters (LAM)* and *Mexicans at Uchicago (UCMA)* at the University of Chicago. Some of the events we like to organize are:
```
Comment (shharrison) [inline README.md:5]: Good use of bold, strikethrough, and italics.

File: README.md [hunk 3]
```diff
@@ -1 +1,16 @@
...
+
+- Discussions with speakers from around the world 
+- Wine Tasting
+    - Includes some games to meet each other
+- Trips to latinamerican countries
+
```
Comment (shharrison) [inline README.md:11]: Good use of bullets and sub-bullets.

File: README.md [hunk 4]
```diff
@@ -1 +1,16 @@
...
+#### Biography (HW1) 
+
+Hi, 
+
+My name is **Victor Vazquez**. I am a second year MPP student from ~~Haris Schul~~, sorry, from Harris School of Public Policy. I have work experience in the private sector, especially in the banking sector. I am a current member of the organizations *Latin American Matters (LAM)* and *Mexicans at Uchicago (UCMA)* at the University of Chicago. Some of the events we like to organize are:
+
+- Discussions with speakers from around the world 
+- Wine Tasting
+    - Includes some games to meet each other
+- Trips to latinamerican countries
```
Comment (shharrison) [inline README.md:10]: Be sure to proofread the text.

File: README.md [hunk 5]
```diff
@@ -1 +1,16 @@
...
+
+One of the links of LAM is [here](https://lamuchicago.wordpress.com/).
+
```
Comment (shharrison) [inline README.md:13]: Good use of links.

File: README.md [hunk 6]
```diff
@@ -1 +1,16 @@
...
+![LAM logo](/Users/victorhugovazquez/Desktop/lam.jpg)
+![Victor](/Users/victorhugovazquez/Desktop/picture.jpg)
+
```
Comment (shharrison) [inline README.md:16]: The images didn't render properly.  I think it's because you used a local file path.

**Review by shharrison — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |           |        0      |            |
| **Coding strategy**         |       +    |              |            |
| **Presentation**            |           |              |         -   |
| **Achievement, creativity** |           |       0       |            |
| **Ease of access**          |     +      |              |            |

Remarks:

* Elaborate on above, especially for "needs work."

   *The images did not render. It would have been best for the assignment's presentation if they were left off the submission.*
  
* Some specific praise?
   
     *The formatting choices Victor made were appropriate and well-executed. I thought that it was clever that he used the assignment to promote his student organization*

* Something I learned?
  
    *I don't think the strikethrough formatting was featured on the provided Markdown style guide, so I learned how to do that.*

* Specific constructive criticism?
   
     *Just make sure to preview your code before submitting it. I think you would have caught the issue with the images if you had.*

* Something I know and that you, my peer, might like to know because it is relevant to something you struggled with.

     *You can preview the code on Github by clicking the "Display the rich diff" button under Pull Requests -> Files changed. I suspect the images rendered correctly in R because they were local pathways but if you had previewed them on Github, you could have caught the issue.*

**Review by msblanders — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |          x |              |            |
| **Coding strategy**         |       x    |              |            |
| **Presentation**            |           |             x |            |
| **Achievement, creativity** |           |            x  |            |
| **Ease of access**          |        x   |              |            |

Remarks:


I thought it was really solid overall. There were some spelling mistakes, and I think the two images (one of Victor, one of the logo) didn't work properly. But other than that, it covered the ground of the assignment well, with lists and headers and italics and bold. Very nice job!

File: README.md (hunk 6 above)
Comment (zxgneil) [reply README.md:16]: Yes, that is why the pictures file to load. Try using a relative file path next time.

**Comment by zxgneil** [comment]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |           |       x       |            |
| **Coding strategy**         |           |      x        |            |
| **Presentation**            |           |       x       |            |
| **Achievement, creativity** |           |      x        |            |
| **Ease of access**          |           |       x       |            |

Remarks:
1. Good experimenting with more than 4 aspects of markdown.
1. The commit history is too simple. Try commit more next time to prevent accidents/mistakes. 
1. The local image does not show correctly because you are using an absolute file path. Try using a relative file path next time!

**Pull request closed without merging**